src/test/regress/expected/failure_failover_to_local_execution.out

```diff
@@ -68,15 +69,17 @@ CONTEXT:  while executing command on localhost:xxxxx
 -- if the local execution is disabled, Citus does
 -- not try to fallback to local execution
 SET citus.enable_local_execution TO false;
-SELECT citus.mitmproxy('conn.delay(500)');
+SET citus.node_connection_timeout TO 900;
+SELECT citus.mitmproxy('conn.connect_delay(1400)');
  mitmproxy
 ---------------------------------------------------------------------
 
 (1 row)
 
 SET citus.log_local_commands TO ON;
 SELECT count(*) FROM failover_to_local;
-ERROR:  could not establish any connections to the node localhost:xxxxx after 400 ms
+ERROR:  could not establish any connections to the node localhost:xxxxx after 900 ms
```

Comment:
does 900ms make it more stable? Is that why we do the changes in this file?

Reply:
Alright, I reverted the timing changes, but kept the RESET (and the new function name). If this test ever becomes flaky we con try to increase the timeouts here too.